pages/home.tsx

```diff
@@ -21,8 +24,21 @@ export function Home(props: HomeProps): JSX.Element {
         <title>Home | TipTracker</title>
       </Head>
 
-      <h1>Home</h1>
-      <h2>Welcome back: {session?.user?.name || "Guest"}</h2>
+      {session ? (
```

Comment:
This is just for the sake of demo. This page will be updated to match the mockup when we have more functionality.